synthetics-client/main.py

```diff
@@ -66,8 +68,10 @@ def cli(
             ca_certs=ca_certs_path
     )
 
-    Scheduler(client)
+    Scheduler(client, ip_info_access_token)
```

Comment:
why exactly do we use a separate python script instead of doing this in OpenSearch plugin? any reference for this?

Reply:
Will link the design doc here in a bit. That would make things clearer.